pkg/backend/backend.go

```diff
@@ -25,11 +26,21 @@ type Server struct {
 	pb.UnimplementedNVMfRemoteControllerServiceServer
 	pb.UnimplementedNullDebugServiceServer
 	pb.UnimplementedAioControllerServiceServer
+
+	rpc server.JSONRPC
 }
 
 // NewServer creates initialized instance of BackEnd server
 func NewServer() *Server {
-	return &Server{}
+	return NewServerWithJSONRPC(server.DefaultJSONRPC)
+}
+
+// NewServerWithJSONRPC creates initialized instance of BackEnd server communicating
```

Comment:
what is the plan here ? deprecate on of them in next PR ? or you want to keep both... ?

Reply:
discussed in slack